CHANGELOG.md

```diff
@@ -1,6 +1,155 @@
+## 1.18.0 (2020-02-06)
+
+This is a feature release with quite a few new features and fixes.
+
+It includes new flags for `fluxd` and `fluxctl`; namely, it includes a new
+flag to disable registry scanning completely  (`--registry-disable-scanning`)
+which allows deploying Flux without memcached.
+
+There is a new `.flux.yaml` variant (`scanForFiles`) which allows telling
+Flux to scan the local files, which is useful when mixing
+`--manifest-generation` with raw manifests.
+
+This release also includes a few bugfixes. Namely, it comes with a fix for a
+filesystem leak in which git clone mirrors weren't being removed.
+
+### Enhancements
+
+- Disable Image Scanning with `--registry-disable-scanning`
+  fluxcd/flux{[#2745][fluxcd/flux#2745], [#2753][fluxcd/flux#2753]
+  [#2798][fluxcd/flux#2798], [#2813][fluxcd/flux#2813]}
+- Add `scanForFiles` variant of `.flux.yaml` to scan current directory
+  for manifests instead of generating them [fluxcd/flux#2638][]
+- Honor KUBECONFIG env variable in fluxd [fluxcd/flux#2741][]
+  [fluxcd/flux#2760][]
+- Make Kubernetes resource-exclusion configurable through
```

Comment:
Something went wrong here with the formatting.

Reply:
I don't follow

Reply:
It should be `fluxcd/flux{[#2749][fluxcd/flux#2749], [#2754][fluxcd/flux#2754]}`

Reply:
Ah, I was confused by the line addressed by the comment.

Reply:
Fixed!